lib/experimental/class-gutenberg-rest-block-form-submit-controller.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Block_Form_Submit_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ * @since      6.3.0
+ */
+
+/**
+ * Core class used to submit forms (using the form block) via the REST API.
+ *
+ * @see   WP_REST_Controller
+ */
+class Gutenberg_REST_Block_Form_Submit_Controller extends WP_REST_Controller {
+
+	/**
+	 * The namespace of this controller's route.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wp/v2';
+
+	/**
+	 * The base of this controller's route.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'block-form-submit';
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @see   register_rest_route()
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => 'POST',
+					'callback'            => array( $this, 'form_submit' ),
+					'permission_callback' => array( $this, 'form_submit_permissions_check' ),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Checks whether a given request has permission to read block patterns.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 *
+	 * @return WP_Error|bool True if the request has read access, WP_Error object otherwise.
+	 */
+	public function form_submit_permissions_check( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		return rest_cookie_check_errors( null );
+	}
+
+	/**
+	 * Retrieves all block pattern categories.
```

Comment:
Is this comment here accurate?

Reply:
Definitely not... It was just copy/paste. Fixed it now, thanks for catching this! 👍